spark/src/main/scala/org/apache/spark/sql/comet/operators.scala

```diff
@@ -81,7 +81,11 @@ abstract class CometExec extends CometPlan {
 
       out.flush()
       out.close()
-      Iterator((count, cbbos.toChunkedByteBuffer))
+      if (out.size() > 0) {
```

Comment:
Oh, I see. The compressed stream is wrapped in `out`. When it is flushed, it might write compression related bytes which are not counted in `out.size()`.

Reply:
I think that is same.